cogs/utils/dataIO.py

```diff
@@ -9,6 +9,15 @@ class InvalidFileIO(Exception):
 class DataIO():
     def __init__(self):
         self.logger = logging.getLogger("red")
+        try:
+            __import__("aiofiles")
+            import aiofiles
+            self.async_compatible = True
+        except ImportError:
+            self.logger.exception("Attempted to use aiofiles but "
```

Comment:
I wrote it that way knowing from my time watching MusicBot users update their bots by downloading and dragging the bot folder into the main install directory and not actually updating their deps file. This ensures that a deps change wont be the reason the bot breaks and can expect this code to be removed once most are back up to date.